### ISSUE TITLE
[Error Report] RecNN on Colab isn't working

### ISSUE THREAD

**Issue by Rowing0914** [awarebayes/RecNN#19]
Thank you for your contribution to the community.
I'd like to remind you of the error I encountered just now.
It seems that the [current your notebook](https://colab.research.google.com/drive/1xWX4JQvlcx3mizwL4gB0THEyxw6LsXTL) isn't working. Please check it once when you have time.

![Screenshot from 2020-12-06 21-14-35](https://user-images.githubusercontent.com/20626792/101279829-5a407680-3808-11eb-9ccd-c83faf74fa04.png)
![Screenshot from 2020-12-06 21-14-51](https://user-images.githubusercontent.com/20626792/101279830-5ad90d00-3808-11eb-8c85-ea994442820f.png)

**Comment by awarebayes** [comment]
Sorry, didn't get the email notification in time
Working on it

**Comment by Rowing0914** [comment]
> Sorry, didn't get the email notification in time
> Working on it

Thank you!! No hurry! Please take your time and looking forward to any update!

**Comment by awarebayes** [comment]
![Screenshot](https://user-images.githubusercontent.com/42784580/101480205-69d4d200-3964-11eb-8f84-a7e961635009.png)

**Comment by awarebayes** [comment]
TL;DR it preemptively crashes session, it is needed after pandas/ray installation
All is documented and written in plain text
